packages/coin-tester/src/utils/componentMaker.tsx

```diff
@@ -99,6 +99,7 @@ export const TwoInputs = (input: {
               input.setNumberValue(value ? value : 0);
             }
           }}
+          style={{ wordBreak: 'break-words' }}
```

Comment:
Unrelated change please use `packages/coin-tester/src/components/Inputs.tsx` instead.

Reply:
i will update it